keda.yaml

```diff
@@ -1,13 +1,16 @@
 package:
   name: keda
```

Comment:
Do you want to version stream this?

Reply:
otherwise the provides here are not necessary.

Reply:
This is where it's a little unclear to me. Yes I want to version stream it, but I also want it to track latest. So I don't want to call it `keda-2.12` since it will eventually switch to `2.13` or `3.0` right?

Reply:
So this is the "latest version stream"

Reply:
Yeah, it just makes it more complicated in images-private